aries-core/src/js/components/external/stories/TextArea.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Grommet, Box, FormField, TextArea } from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'TextArea',
+};
+
+export const Simple = () => (
+  <Grommet theme={aries} full>
+    <Box align="center" pad="large">
+      <Box width="medium">
+        <FormField label="Additional feedback" htmlFor="text-area-example">
```

Comment:
This is more of a FormField example rater than TextArea

Reply:
I think for accessibility, TextArea is paired with FormField because if there is no associated label, it won't pass accessibility checks. I pulled this example from what we have on the design system site. Open for discussion on what this story is, but for all of these I can just re-used what we have on the site.